examples/7guis/cells.slint

```diff
@@ -58,15 +58,16 @@ export component MainWindow inherits Window {
                 }
 
                 l := LineEdit {
-                    visible: is-active;
-                    width: 100%;
-                    height: 100%;
                     edited => {
                         cell = { value: self.text.to-float(), formula: self.text };
                     }
                     accepted => {
                         root.active-cell = { r: -1, c: -1};
                     }
+
+                    visible: is-active;
+                    width: 100%;
+                    height: 100%;
```

Comment:
not sure about this part of the guidelines that put callbacks connection before properties

Reply:
I think that's consistent with properties the rest of the callbacks and function definitions.

Reply:
<img width="770" alt="image" src="https://github.com/slint-ui/slint/assets/6715107/29792c95-f2f1-41ab-a0bc-7062e7511ce8">